### ISSUE TITLE
Ambiguous toggle switches?

### ISSUE THREAD

**Issue by martin-nc** [getgrav/grav-plugin-admin#1975]
Just trying Grav for the first time, and here's a quick question. Is Markdown extra turned on or off in this image:
![Screenshot 2020-11-17 at 22 57 35](https://user-images.githubusercontent.com/15261745/99461040-ee659f00-2928-11eb-820d-e5d59fa1d7a1.png)
I genuinely don't know. The slightly grey text of Yes and pale background suggests 'No' is activated. However when you click 'Yes' you get a grey background:
![Screenshot 2020-11-17 at 23 06 42](https://user-images.githubusercontent.com/15261745/99461359-98ddc200-2929-11eb-9294-78c6fe8e6cf2.png)
Grey doesn't suggest active to me. Now I've no idea of what is active. It doesn't help that the words are 'Yes' and 'No' rather than 'On' and 'Off'. Here is an example of an unambiguous scheme grabbed randomly from the web. It uses more common colour conventions and only shows you the option that is active:
![Screenshot 2020-11-17 at 23 00 01](https://user-images.githubusercontent.com/15261745/99461553-e823f280-2929-11eb-9d4a-512e73ad5b66.png)
Perhaps I can suggest making the buttons less ambiguous?

EDIT: to give a quick idea what I mean, even without a change in colour:
![toggle-buttons2](https://user-images.githubusercontent.com/15261745/99464637-7ef3ad80-2930-11eb-9c97-1c33a358480e.png)

**Comment by jonampro** [comment]
Hi, usually Yes = 1, No = 0. There is an extra highlight option for toggles in blueprints. I suppose the logic behind this is, that you can mark the recommended setting this way.
<img width="265" alt="Bildschirmfoto 2020-11-18 um 18 09 04" src="https://user-images.githubusercontent.com/47760171/99563927-1527e200-29ca-11eb-8947-139c90a3bb6e.png">

**Comment by rhukster** [comment]
I appreciate what you guys are saying, however, there are specific reasons for this.  

1. the buttons are not always yes/no, there are many situations where the toggle I used between two arbitrary text strings, where there is no right/wrong.
2. there can be multiple options, 3 or  even 4, again, the color would be a problem here
3. we have a concept for a 'highlight' option that is used to indicate the 'default' or preferred option.  That is what gives the buttons the blue color.  The default might well be 'No', so highlighting that blue shows that it's set and is the default.  Changing a toggle to yes in that case, shows that you have set the non-default or non-preferred option.
4. everything is now themeable in admin for 1.10, and the colors of the toggles now change with the color scheme.

In short, the toggle is flexible, and can be used in many ways, so forcing the colors is only going to reduce functionality and flexibility.
